### ISSUE TITLE
Multilingual Captions

### ISSUE THREAD

**Issue by nicolas-raoul** [commons-app/apps-android-commons#2096]
> Multilingual Captions, the first feature release for Structured Data, is coming in January of 2019

--- https://meta.wikimedia.org/wiki/Structured_Data_on_Commons/Newsletter/Next

My understanding is that it is like a bit like a filename, but multilingual, and limited to 255 characters. More at https://www.mediawiki.org/wiki/Help:File_captions

The underlying idea is that filenames are risky to modify (all links break at each typo correction) and they are not translatable, so when showing a picture the software would show this caption under it, rather than the filename (which might be in Swahili) or the description (which might be a huge wall of text).

# Example

- Now: I take a picture of Obama in Tibet, and upload it with the title "_Obama in Nepal_" and the English description "_I randomly stumbled upon  Barrack Obama in Solma (Nepal) in the afternoon of December 10, 2018, after his speech at the Eco Summit 2018._"
- After: I take a picture of Obama in Tibet, and upload it with the English caption "_Obama in Nepal_" and the English description "_I randomly stumbled upon  Barrack Obama in Solma (Nepal) in the afternoon of December 10, 2018, after his speech at the Eco Summit 2018._"

# Implementation

Captions will be optional at first, but as most new files will have them from January 2019, our uploads may start to stick out if none have them.

I suggest we just:

1. Rename the current filename field to "Caption"
2. Make it multilingual
3. When uploading, just take the first caption and use it as the filename

**Comment by nicolas-raoul** [comment]
This Commons feature will be released either today or tomorrow.
I believe this is rather easy to implement for a newcomer, by seeing how it is done with the description field, then adding a parameter to the Commons upload request.

**Comment by nicolas-raoul** [comment]
Duplicate of #2297
